sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5921,6 +5921,8 @@ pi_result _pi_queue::synchronize() {
     return PI_SUCCESS;
 
   // For in-order queue just wait for the last command.
+  // If event is discarded then it can be in reset state or underlying level
+  // zero handle can have device scope, so we can't synchronize the last event.
```

Comment:
```suggestion
  // zero handle may not have device scope, so we can't synchronize the last event.
```

Reply:
If it has device scope then we can't synchronize, right? And if it has host scope then we can synchronize.

Reply:
Right, I read "device scope" as "host scope" for some reason. Nevermind.